Breeding-Insight/bi-web
# [BI-1427] - Remove unused Traits Vue Components

# Description
**Story:** [BI-1427] - Remove unused Traits Vue Components

_Please include a summary of the change that was made._



# Dependencies
bi-api/develop

# Testing
- Visit ontology related pages
- Ensure path is ontology and not trait path

# Checklist:
